Sources/XCHammer/XcodeTarget.swift

```diff
@@ -1075,20 +1075,22 @@ public func makeXcodeGenTarget(from xcodeTarget: XcodeTarget) -> ProjectSpec.Tar
         .map { ProjectSpec.Dependency(type: .target, reference: $0.xcTargetName,
                 embed: $0.isExtension) }
 
-    // Fuse or use the actual rule entry
+    // Determine settings, sources, and deps
+    // There are 2 different possibilities:
+    // 1) High level "Fusing" of a target from multiple targets
+    // 2) A simple conversion of the target: taking it "as is"
     if shouldFlatten(xcodeTarget: xcodeTarget) {
-        let unwrapedFlatFlattendDeps = xcodeTarget.unfilteredDependencies
-            .filter { flattened.contains($0) }
-        // Source sources of unwrapped flattend deps,
-        // assuming these sources' are the source's source of truth.
-        sources = unwrapedFlatFlattendDeps.flatMap { $0.xcSources }
+        // Determine deps to fuse into the rule.
+        let fusableDeps = xcodeTarget.unfilteredDependencies
```

Comment:
RIP unwrapedFlatFlattendDeps

Reply:
Not sure if there is a better way to describe this :trollface: